website/content/blog/2024-12-01-a-year-in-review.md

```diff
@@ -0,0 +1,106 @@
+  * [Implement the Direct3D 12 / GDK/GDKX backend \#8195](https://github.com/MonoGame/MonoGame/issues/8195)
+
+* #### In Progress Bounties
+
+  * [Switch the console runtime from BRUTE to NativeAOT \#8194](https://github.com/MonoGame/MonoGame/issues/8194)  
+  * [A better 2D onboarding tutorial \#8317](https://github.com/MonoGame/MonoGame/issues/8317)  
+  * [A better 3D onboarding tutorial \#8318](https://github.com/MonoGame/MonoGame/issues/8318)
+
+* #### More bounties to come in the areas of:
+
+  * Graphics  
+    * Optimise and fix DesktopVK  
+      * Why: it is still experimental and there is some unexpected flickers  
+  * Content  
+    * Native Texture Compression and Asset processing shared object  
+      * Why: enhance assets processing performance and reduce dependencies  
+    * Upgrade Shader Compiler to use [https://github.com/microsoft/ShaderConductor](https://github.com/microsoft/ShaderConductor) and [https://github.com/KhronosGroup/SPIRV-Cross](https://github.com/KhronosGroup/SPIRV-Cross)	  
+      * Why: retiring mojoshader to allow future improvements on shaders  
+  * Audio  
+    * FAudio implementation for DesktopVK  
+      * Why: replace OpenAL-Soft which can’t be totally XAudio-accurate  
+  * Tutorials and content  
+    * Advanced 3D concepts (Some ideas I know this isn’t a planning doc)  
+      * Custom Effects   
+      * Culling  
+      * Shadows  
+      * Post Processing  
+      * Scene Management?  
+      * Deferred Rendering  
+      * More starter kits for 2D and 3D  
+      * More shader guidance/samples
+
+* #### Bug Fixes and Enhancements
+
+  * Re-enabled the Unit Tests for PR’s.  
+  * [Upgrade Mac Editor to net8.0-macos. Upgrade Eto.Forms](https://github.com/MonoGame/MonoGame/pull/8505)   
+  * Fix Issues with OpenAL Panning on DesktopGL and Mobile  [https://github.com/MonoGame/MonoGame/pull/8480](https://github.com/MonoGame/MonoGame/pull/8480) and [https://github.com/MonoGame/MonoGame/pull/8466](https://github.com/MonoGame/MonoGame/pull/8466)
+  * [Detect GamePad/Keyboard events on Android](https://github.com/MonoGame/MonoGame/pull/8465)
+  * [Remove string allocations when calling GamePad.GetCapabilities](https://github.com/MonoGame/MonoGame/pull/8453)
+  * [Content Pipeline now works on M1/M2 Macs](https://github.com/MonoGame/MonoGame/pull/8570)
+  * [Fix Android MediaPlayer Song implementation](https://github.com/MonoGame/MonoGame/pull/8583)
+  * [Update MonoGame.Library.OpenAL to 1.23.1.10](https://github.com/MonoGame/MonoGame/pull/8560)
+  * [Fix Sprite font multi byte parsing](https://github.com/MonoGame/MonoGame/pull/8554)
+  * [Added Vibration and Caps to Native GamePad API](https://github.com/MonoGame/MonoGame/pull/8520)
+  * [Fix MacOS Content directory location](https://github.com/MonoGame/MonoGame/pull/8479)
+  * [Detect GamePad/Keyboard events on Android](https://github.com/MonoGame/MonoGame/pull/8465)
+
+* #### Game Released (that we know of)
```

Comment:
Don't forget to look through the 99 games released this year on itch.io tagged "made-with-monoGame":
https://itch.io/games/newest/made-with-monogame

Not seen or played these but some look interesting:
[Breeze & Freeze](https://msiebenmann.itch.io/breeze-and-freeze)
[Tic Tac Dough](https://ms00.itch.io/tic-tac-dough)
[Ninja Cat Remewstered](https://pixelshock.itch.io/ninja-cat-remewstered)
[Smack 'n' Snatch](https://skyyyla.itch.io/smack-n-snatch)
[FishGrid](https://shmellyorc.itch.io/fishgrid)

Also random off topic idea, put together a MonoGame [Co-op Bundle](https://itch.io/docs/creators/bundles) of selected paid games (a modest cut for raising MonoGame donations if participants agree maybe).

Reply:
@squarebananas Didnt' realise I could do that! Thanks!!

Reply:
@squarebananas We were already talking internally about doing a bundle. Hopefully we can get something organised in the new year.

Reply:
Added the ones mentioned above. I unfortunately don't have enough time to go through more before getting this post out.  We can always add more later :)

Reply:
Sounds good. Just to reiterate I've "not seen or played" those games, they were just the ones that stood out to me. I hope they're alright, they all have positive player comments at least.